### PR TITLE
Release 2.0-patch.3

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 * Fixes
 
+## 2.0-patch.3
+
+* Fixes
+
+  * Reverted https://github.com/dcos/dcos-core-cli/pull/322
```
Comment (bamarni) [inline CHANGELOG.md]: A changelog targets users, here as a user I wouldn't understand this fix, even though I follow the PR link.

```suggestion
  * Fixed creation of nested groups, the regression was introduced in #322
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
+  * Fixed integration test for cosmos changes (#368)
```
Comment (bamarni) [inline CHANGELOG.md]:
```suggestion
```

Usually we only put user-facing changes in the changelog.

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes @bamarni @rgo3  Can you review, please?

**Review by drewvanstone — APPROVED** [review]

**Review by makkes — APPROVED** [review]

**Pull request merged**